.ci/macos-steps.yaml

```diff
@@ -14,7 +14,7 @@ steps:
     set -e
     sudo xcode-select --switch /Applications/Xcode_10.1.app/Contents/Developer
     unset BOOST_ROOT
-    pip install cython numpy pandas zipp
+    pip install cython numpy pandas zipp configparser
```

Comment:
I don't have a problem adding this, but are you sure it's needed?

Reply:
I think so this is already added I don't why this is happening after revert my local changes.
I am really sorry about this but this is resolved after rebasing the branch.